remote_rest/main.py

```diff
@@ -0,0 +1,168 @@
+from contextlib import asynccontextmanager
```

Comment:
We might talk about extracting that attackmate-remote-server to a seperate repo. People who use attackmate in their python-scripts, might not want to download all the dependencies for the attackmate-remote-server